azure-mgmt-scheduler/src/main/java/com/microsoft/azure/management/scheduler/JobScheduleMonthlyWeekDay.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.scheduler;
+
+import com.microsoft.azure.management.apigeneration.Beta;
+import com.microsoft.azure.management.apigeneration.Fluent;
+
+/**
+ * The day of the week the job will run each month on weekly bases.
```

Comment:
maybe a little more accurate/clearer:
"Days of the week relative to the beginning of the month for scheduling a job."